R/utilities-mpi-error-handling.R

```diff
@@ -44,9 +53,34 @@ loadSimulationOnCores <- function(structureSet, logFolder) {
   }
 }
 
+
+
+
+#' @title loadPopulationOnCores
+#' @description Send population file names to cores, check that population is loaded on each core successfully
+#' @param structureSet containing simulationSet which contains path to simulation file and pathIDs to be loaded in simulation object as outputs
+#' @param logFolder folder where the logs are saved
+loadPopulationOnCores <- function(populationFiles, logFolder) {
+  Rmpi::mpi.bcast.Robj2slave(obj = populationFiles)
+  Rmpi::mpi.remote.exec(population <- NULL)
```

Comment:
Why this line? You do not need to instantiate to overwrite immediately no?

Reply:
removed